app/controllers/hosts_controller.rb

```diff
@@ -196,8 +196,8 @@ def externalNodes
 
     begin
       respond_to do |format|
-        format.html { render :text => "<pre>#{ERB::Util.html_escape(@host.info.to_yaml)}</pre>" }
-        format.yml { render :text => @host.info.to_yaml }
+        format.html { render :text => "<pre>#{ERB::Util.html_escape(@host.info.to_yaml(:line_width => -1))}</pre>" }
+        format.yml { render :text => @host.info.to_yaml(:line_width => -1) }
```

Comment:
i would suggest extracting the yaml into variable to reduce code duplicities. Good to merge after that.